tools/src/OpenSearchHttpClient.ts

```diff
@@ -64,6 +64,7 @@ export interface AwsAuth {
 
 export interface OpenSearchHttpClientOptions {
   url?: string
+  distribution?: string,
```

Comment:
We also don't need these other changes here now as it's not actually used by `OpenSearchHttpClient`

Reply:
Updated.